include/boost/geometry/io/wkt/write.hpp

```diff
@@ -62,22 +62,63 @@ namespace boost { namespace geometry
 namespace detail { namespace wkt
 {
 
+template <typename Output>
+struct output_formatter
+{
+    template <typename T>
+    static void add(Output & out, T const& v)
```

Comment:
Perhaps [put](https://en.cppreference.com/w/cpp/io/basic_ostream/put) is a better name, than `add`.
Alternatively, `append` if it should work for both, streams and buffers like `string`. What's your opinion, @awulkiew ?

Nitpick: remove whitespaces before `&`. Try to replicate existing code style.

Reply:
@mloskot yes, `append` is a better name, thanks.